lib/streams/stringifier-stream.js

```diff
@@ -22,6 +22,8 @@ module.exports = class StringifierStream extends stream.Transform {
         }
         let st = this.queue.shift();
         if (st instanceof stream) {
+            st.emit('plugged');
```

Comment:
would only emit this if its a instance of async stream though. Totally useless for other fragment streams.

Reply:
Currently yes, but I didn't want to mix up knowledge of async stream into the stringifier stream

Reply:
In that case we would need alternative way of handling for this particular instance.